### ISSUE TITLE
Search Not Working

### ISSUE THREAD

**Issue by Reldey** [OrganicIrradiation/ratebeer#46]
Can't get search to work, only returns empty lists. Is search currently working for others?

Thanks.

**Comment by neur0nic** [comment]
Did you install the package from pip. The problem is solved, but you need to install it from git. (Issue #28 )

```
pip install git+git://github.com/alilja/ratebeer/ratebeer.git
```

**Comment by Reldey** [comment]
I did install it from pip, I am getting a file not found error however using your command. Not sure what I am doing wrong.

**Comment by Reldey** [comment]
Ahh I see, I needed to install git. Could the instructions be updated to reflect the new install process?

**Comment by neur0nic** [comment]
I think @parryc wanted to do something about the pip repository?!

**Comment by parryc** [comment]
@neur0nic, thanks for reminding me – will make another issue for us to discuss with @OrganicIrradiation.

**Comment by parryc** [comment]
Closed, see #47.
